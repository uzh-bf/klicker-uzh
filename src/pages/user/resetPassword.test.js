/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import ResetPassword from './resetPassword'

describe('Snapshot-Testing', () => {
  it.skip('Works', () => {
    const component = renderer.create(<ResetPassword />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
