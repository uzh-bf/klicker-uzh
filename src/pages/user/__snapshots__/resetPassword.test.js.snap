// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div
  className="staticLayout"
  data-jsx={2161656035}
>
  <div
    className="content"
    data-jsx={2161656035}
  >
    <div
      className="login"
      data-jsx={1688187079}
    >
      <h1
        data-jsx={1688187079}
      >
        <span>
          Reset password
        </span>
      </h1>
      <form
        className="ui form error"
        data-jsx={2063282003}
      >
        <div
          className="required field"
        >
          <label
            data-jsx={1632341904}
            forHtml="email"
          >
            Email
          </label>
          <input
            data-jsx={1632341904}
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            onDragStart={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            placeholder="Email"
            type="email"
            value=""
          />
        </div>
        <div
          className="actionArea"
          data-jsx={2063282003}
        >
          <button
            className="ui primary disabled button semanticButton"
            disabled={true}
            onClick={[Function]}
            role="button"
            tabIndex={-1}
            type="submit"
          >
            Submit
          </button>
          <div
            className="links"
            data-jsx={2063282003}
          >
            <div
              className="ui list"
              role="list"
            >
              <div
                className="item"
                data-value={undefined}
                onClick={[Function]}
                role="listitem"
              >
                <a
                  data-jsx={2063282003}
                  href="/user/login"
                  onClick={[Function]}
                >
                  Back to login
                </a>
              </div>
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>
  <footer
    data-jsx={2161656035}
  >
    Â© IBF
  </footer>
</div>
`;
